tool/travis.sh

```diff
@@ -134,9 +134,11 @@ elif [ "$BOT" = "test_ddc" ]; then
     # Run every test except for integration_tests.
     # The flutter tool doesn't support excluding a specific set of targets,
     # so we explicitly provide them.
-    flutter test test/*.dart test/{core,fixtures,flutter,support,ui}/
-    flutter test --platform chrome test/*.dart test/{core,fixtures,flutter,support,ui}/
-
+    if [ "$PLATFORM" = "vm" ]; then
```

Comment:
looks good as is

You could also have an explicit `chrome` branch (in addition to the `vm` one), and have the else clause fail.

Reply:
Done.